fsgs/Database.py

```diff
@@ -439,18 +439,11 @@ def get_variant_for_list_and_game(self, list_uuid, game_uuid):
             return None
         return row[0]
 
-    def find_games_new(self, search="", have=3, list_uuid="",
+    def find_games_new(self, search="", have=3, list_uuid=None,
                        database_only=False):
-        print("Database.find_games_new search = {0}".format(repr(search)))
-        non_database_only = False
-        if list_uuid == self.GAME_LIST_GAMES:
-            database_only = True
-            list_uuid = ""
-        elif list_uuid == self.GAME_LIST_CONFIGS:
-            non_database_only = True
-            list_uuid = ""
-        elif list_uuid:
-            have = 0
+
+        # FIXME: game.game_list_game never populated?
+        ENABLE_JOIN = False
```

Comment:
I didn't understand whether we are using table `game.game_list_game` or not.

I have it always empty in my fs-uae-launcher installation, so I'm not sure this JOIN is needed or not: could you clarify this?